pl/basics/pattern-matching.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: Dopasowanie wzorców
+category: basics
+order: 4
+lang: pl
+---
+
+Dopasowanie wzorców to jedno z najmocniejszych elementów Elixira pozwala dopasować wartości, struktury danych, a nawet funkcje. Tę lekcję rozpoczniemy od przyjrzenia się jak używać tego mechanizmu.
```

Comment:
Something doesn't suit here very well. I think about changing the first sentence to:

> Dopasowanie wzorców, to jeden z najmocniejszych elementów Elixira, który pozwala dopasować wartości, struktury danych, a nawet funkcje.